### PR TITLE
Update docs to avoid installing grunt-cli

### PR DESCRIPTION
**Pull request by nekonenene**
## Proposed changes

Developers don't need to install grunt-cli because we already installed into node_modules via `npm install`. Now, I replaced commands in docs using `npm` instead of `grunt`.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist:

- [x] I have read the **[CONTRIBUTING document](https://github.com/materializecss/materialize/blob/master/CONTRIBUTING.md)**.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by Smankusors — APPROVED** [review]
this seems better IMO, what do you think? @DanielRuf

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ If you want to help us translate the documentation into other languages, please
 - [Example Jasmine Tests](https://github.com/materializecss/materialize/tree/master/tests/spec)
 - [Travis CI](https://travis-ci.org/materializecss/materialize)
 
-Before you start, make sure you install grunt and all its dependencies. To verify you have all the correct dependencies you can run `grunt travis` and it will run the tests. If you get an errors and have not made any changes, it means you have not installed the proper dependencies.
+After `npm install`, you can run `npm test` and it will run the tests. If you get an errors and have not made any changes, it means you have not installed the proper dependencies.
```
Comment (DanielRuf) [inline CONTRIBUTING.md:167]: an errors => any errors

**Review by DanielRuf — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (nekonenene) [reply CONTRIBUTING.md]: Thanks! Fixed.

**Review by DanielRuf — APPROVED** [review]

**Comment by DanielRuf** [comment]
Thanks, this is much better than before.

**Comment by DanielRuf** [comment]
@nekonenene you should have received an invitation for the materializecss org and to the members team of it.

You can see and accept it at https://github.com/materializecss

**Comment by nekonenene** [comment]
@DanielRuf 
Thanks! I joined. 💪

**Pull request merged**